### PR TITLE
Hisban db setup

### PR DESCRIPTION
**Pull request by hisbanbardai**

### CODE REVIEW

File: backend/db/seeds/recipes.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  recipes (
+    user_id,
+    cuisine_id,
+    diet_id,
+    meal_type_id,
+    title,
+    image,
+    prep_time,
+    instructions,
+    protiens,
```
Comment (eobda) [inline backend/db/seeds/recipes.sql:11]: typo - should be `proteins`

File: backend/db/schema/recipes.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    id SERIAL PRIMARY KEY NOT NULL,
+    user_id uuid REFERENCES users (id) ON DELETE CASCADE,
+    cuisine_id INTEGER REFERENCES cuisines (id) ON DELETE CASCADE,
+    diet_id INTEGER REFERENCES diets (id) ON DELETE CASCADE,
+    meal_type_id INTEGER REFERENCES meal_types (id) ON DELETE CASCADE,
+    title VARCHAR(255) NOT NULL,
+    image TEXT NOT NULL,
+    prep_time INTEGER NOT NULL,
+    instructions TEXT NOT NULL,
+    protiens VARCHAR(255),
```
Comment (eobda) [inline backend/db/schema/recipes.sql:17]: typo - should be `proteins`

File: backend/db/seeds/recipes.sql [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
```
Comment (eobda) [inline backend/db/seeds/recipes.sql:48]: typo - should be `proteins`

**Review by eobda — CHANGES_REQUESTED** [review]
* change all instances of `protiens` to `proteins`
* capitalize cuisine names

File: backend/db/seeds/recipes.sql (hunk 1 above)
Comment (hisbanbardai) [reply backend/db/seeds/recipes.sql]: Fixed

File: backend/db/schema/recipes.sql (hunk 2 above)
Comment (hisbanbardai) [reply backend/db/schema/recipes.sql]: Fixed

File: backend/db/seeds/recipes.sql (hunk 3 above)
Comment (hisbanbardai) [reply backend/db/seeds/recipes.sql]: Fixed.

File: backend/db/seeds/cuisines.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO
+  cuisines (name)
+VALUES
+  ('french');
```
[the remark this answers is not in the document]
Comment (hisbanbardai) [reply backend/db/seeds/cuisines.sql]: Fixed.

**Review by eobda — APPROVED** [review]

**Pull request merged**